taar/adapters/dynamo.py

```diff
@@ -0,0 +1,53 @@
+import boto3
+import json
+
+
+class ProfileController(object):
+    """
+    This class
```

Comment:
Can you explain what is this class about and what's supposed to do in this docstring?

Reply:
Doh.  That was bad.  I've updated the docstring now, but this class just provides basic r/w access into a DynamoDB backed datastore.